cpp/ql/src/Critical/DoubleFree.qhelp

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>
+This rule finds memory blocks that are freed more than once. Freeing a
+memory block twice may result in memory corruption for example if the
```

Comment:
I'm not a big fan of the wording here, currently. But let's leave that to the docs team.